pkg/netpol/connlist/exposure_map.go

```diff
@@ -43,14 +53,15 @@ func (ex *exposureMaps) addPeerUnprotectedData(pe *eval.PolicyEngine, peer Peer,
 		return false, err
 	}
 	if !isProtected {
-		ex.addNewEntry(peer, isProtected, isIngress)
+		ex.addNewEntry(peer, false, isIngress)
 		return true, nil
 	}
 	return false, nil
 }
 
 // addPeerXgressEntireClusterExp checks and adds (if exists) ingress/egress entire cluster exposure for the given peer
-// on the given direction
+// on the given direction;
+// each entry in the relevant map may has at-most one connection with entire-cluster (which is added by this func)
```

Comment:
also relevant to update in documentation for `peerXgressExposureData` ? (that there is at most one such object with entire cluster conn ) within the relevant fiield?

Reply:
yes, updated